### PR TITLE
ES6ify videojs-options

### PR DESCRIPTION
**Pull request by philwills**
## Does this affect other platforms - Amp, Apps, etc?
No

## Tested in CODE?
Yes

### CODE REVIEW

**Comment by philwills** [comment]
The automatic processes failed early, so I've done this by hand without the normal trail of commits.

File: static/src/javascripts/projects/common/modules/video/videojs-options.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    // `autoplay` is always set to false.
+    // If you are going to set autoplay to any other value, note it breaks
+    // `preload="auto"` on < Chrome 35 and `preload="metadata"` on old Safari
+    autoplay: false,
+    preload: 'metadata',
+    techOrder: ['html5'],
+    notSupportedMessage: 'This video is no longer available.',
+};
+
+export default function(overrides: Object) {
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/video/videojs-options.js]: default params available in in es6 if you want them:

```js
export default (overrides: Object = {}) => Object.assign({}, defaults, overrides);
```

Comment (philwills) [inline static/src/javascripts/projects/common/modules/video/videojs-options.js]: Yea, I did consider that, but was worried that the semantics would be slightly different if null was passed, but then thinking a bit more, I guess Flow should catch that.

**Review by sndrs — APPROVED** [review]
lgtm!

**Pull request merged**